duckduckgo/Android
# ADS: Bottom Sheet Components

Task/Issue URL: https://app.asana.com/0/0/1202861107912492/f

### Description
Implementation of new bottom sheet components: PromoBottomSheetDialog and ActionBottomSheetDialog

### Steps to test this PR

- Install from this branch
- Go to Settings > App Components Design Preview
- Tap on Dialogs tab and go to Bottom sheet dialogs
- [ ] Check Action Bottom Sheet looks good
- [ ] Check Promo Bottom Sheet looks good

### UI changes
| ActionBottomSheetDialog  | PromoBottomSheetDialog |
| ------ | ----- |
![Screenshot_20221013-100337_DuckDuckGo](https://user-images.githubusercontent.com/20798495/195559846-5b0cfad2-eb85-43d1-9230-ab2f076b00c3.jpg)|![Screenshot_20221013-100323_DuckDuckGo](https://user-images.githubusercontent.com/20798495/195559887-187f7a89-b50e-4a84-a5b7-5bc8ca43df39.jpg)|
